pimd/pim_cmd.c

```diff
@@ -6351,12 +6352,13 @@ static int pim_ssm_cmd_worker(struct pim_instance *pim, struct vty *vty,
 		break;
 	case PIM_SSM_ERR_DUP:
 		vty_out(vty, "%% duplicate config\n");
+		ret = CMD_WARNING;
```

Comment:
just `return CMD_WARNING` is not enough here?

Reply:
following the way it's coded in other sections